plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -348,4 +353,16 @@ private AutoCloseable withTable(String remoteTableName, String definition)
     {
         return withTable(getSession().getSchema().orElseThrow(), remoteTableName, definition);
     }
+
+    private void updateRuleBasedIdentifierMappingFileAndWait(Path file, List<SchemaMappingRule> schemaMappingRules, List<TableMappingRule> tableMappingRules)
+    {
+        try {
+            updateRuleBasedIdentifierMappingFile(file, schemaMappingRules, tableMappingRules);
+            // wait to ensure rules have been refreshed
+            Thread.sleep(2 * ruleRefreshPeriod.toMillis());
```

Comment:
`2*` isn't necessary, is it?

Reply:
To be safe. should be refreshPeriod + some delta

Reply:
Kept 2x, moved sleep to the Utils class.

Reply:
> should be refreshPeriod + some delta

i don't believe it should

> To be safe.

fine, keep it.